HPXMLtoOpenStudio/resources/hpxml.rb

```diff
@@ -970,13 +970,17 @@ def to_oga(doc)
       building = XMLHelper.add_element(hpxml, 'Building')
       building_building_id = XMLHelper.add_element(building, 'BuildingID')
       XMLHelper.add_attribute(building_building_id, 'id', @building_id)
-      if (not @state_code.nil?) || (not @zip_code.nil?)
+      if (not @state_code.nil?) || (not @zip_code.nil?) || (not @time_zone_utc_offset.nil?)
         site = XMLHelper.add_element(building, 'Site')
         site_id = XMLHelper.add_element(site, 'SiteID')
         XMLHelper.add_attribute(site_id, 'id', 'SiteID')
         address = XMLHelper.add_element(site, 'Address')
         XMLHelper.add_element(address, 'StateCode', @state_code, :string, @state_code_isdefaulted) unless @state_code.nil?
         XMLHelper.add_element(address, 'ZipCode', @zip_code, :string) unless @zip_code.nil?
+        if not @time_zone_utc_offset.nil?
+          extension = XMLHelper.add_element(address, 'extension')
+          XMLHelper.add_element(extension, 'TimeZoneUTCOffset', @time_zone_utc_offset, :integer, @time_zone_utc_offset_isdefaulted)
+        end
```

Comment:
I wonder if we should propose a new `TimeZoneUTCOffset` element for the HPXML schema instead. @nmerket, any problem with that?

Reply:
I discussed w/ @nmerket. We agreed to add a `TimeZone` element under [`Building/Site`](https://hpxml.nrel.gov/datadictionary/3.0.0/Building/Site) that can accommodate multiple child elements (e.g., `Name`, `UTCOffset`, etc.). We'll want to document that the UTC offset should be defined based on standard (not daylight savings) time too.

Reply:
Implemented.